tailwind.config.js

```diff
@@ -1,31 +1,31 @@
 // tailwind.config.js
 module.exports = {
-  darkMode: 'class',
+  darkMode: "class",
   content: [
-    './src/**/*.{js,jsx,ts,tsx}',
-    './src/pages/**/*.{js,jsx,ts,tsx}',
-    './src/components/**/*.{js,jsx,ts,tsx}',
-    './src/templates/**/*.{js,jsx,ts,tsx}',
+    "./src/**/*.{js,jsx,ts,tsx}",
+    "./src/pages/**/*.{js,jsx,ts,tsx}",
+    "./src/components/**/*.{js,jsx,ts,tsx}",
+    "./src/templates/**/*.{js,jsx,ts,tsx}",
   ],
   theme: {
     colors: {
-      white: '#ffffff',
-      red: '#cc091c',
+      white: "#ffffff",
+      red: "#8A2A2B",
```

Comment:
looks like we just had 1 red before 🤔 we could add another if we need a more vibrant one though

Reply:
We should still only have/need 1 red? Is that causing an issue somewhere?

Reply:
ah ok, just wanted to check, the new red looked a little brown to me on the dark backgrounds so wanted to be sure 👍